src/rdkafka_sasl_plain.c

```diff
@@ -70,30 +70,56 @@ int rd_kafka_sasl_plain_client_new(rd_kafka_transport_t *rktrans,
                                    size_t errstr_size) {
         rd_kafka_broker_t *rkb = rktrans->rktrans_rkb;
         rd_kafka_t *rk         = rkb->rkb_rk;
+
+        char *username_ptr = rk->rk_conf.sasl.username;
+        char *password_ptr = rk->rk_conf.sasl.password;
+
+        if (rk->rk_conf.sasl.plain_creds_cb) {
+                size_t username_size = 128;
```

Comment:
consistency nit: Here you use 128 bytes, earlier 256 bytes. Any particular reason for that?